b/endless/endless.css

```diff
@@ -70,7 +70,7 @@ h2 {
 	}
 }
 .btn {
-	background: #64DD17;
```

Comment:
Thanks for your contributions! I love seeing people want to contribute to our projects.

So about my comment. I think you mistakenly made the button be **red** when you **have** enough points.

I am guessing that you wanted to update the not enough button's color to the new red color scheme (that is a great idea).

So I like the idea of updating the red in the endless mode selector, even if you did this by accident. Thanks!

So the problem is that you changed the green to the red so when you have enough, it is red, and when you don't have enough, it is the old red.

If you did this purposely, I don't really agree with it because the not enough button still has the outdated red colors. Also, the play button has the red color which in my opinion, symbolizes not enough, or a bad thing like cancel or disabled.

### Solution
So if you want to fix this yourself, that would be wonderful: revert the changes of the background color for `.btn`, and then change the background colors of `#ne` to the new red (what you did to `.btn` in this commit).

If you do not want to do this yourself, I am glad doing it for you.

Thanks a lot!